initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -15,10 +15,9 @@
   <runtime>${RUNTIME_PATH}</runtime>
   <sourcedir>${CONFIG_SOURCE_PATH}</sourcedir>
   <user>${RUNTIME_USER}</user>
-  <ports>
```

Comment:
What was wrong with the old path exactly? Moving it seems like scope for confusion unless there's a good reason